### PR TITLE
Added CDP Resolution Browser Action

### PR DESCRIPTION
**Pull request by ajaysikri**
_A summary of your pull request, including the what change you're making and why._

Adding CDP Resolution browser action. On load of the page, the browser will "fetch" an endpoint on the browser passing in the customer identifier (provided to segment.io customers) and the anonymous ID for the session.

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [X] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

TESTING NOTE: Used the action tester to "send page load" test event.  In the network traffic, the expected call and the parameter values are sent to the endpoint.  Ensured the endpoint was receiving the call and the values in the expected format.

### CODE REVIEW

**Review by joe-ayoub-segment — CHANGES_REQUESTED** [review]
Hi @ajaysikri , 

Thanks for raising this PR and apologies we're only getting to it now. 

Given your PR is for a new Destination I'd like to sync with you to discuss the use-cases for it, and to review the code together. 

The code you've submitted doesn't load a 3rd party library, and only seems to collect the end user's anonymousId, which is unusual. 

Having a discussion should help me better understand what you'd like to accomplish. 

Would you be available to meet? You can schedule a call with me using this Calendly link: 

http://calendly.com/joe_ayoub/integration-workshop

I look forward to hearing from you. 
Regards,
Joe

**Comment by joe-ayoub-segment** [comment]
hi @ajaysikri , just following up on this PR again. Is it something you'd like to proceed with?
Best regards,
Joe

**Comment by ajaysikri** [comment]
Hi Joe,

I have update the code to reflect the changes we discussed.

Thanks, Ajay

File: packages/browser-destinations/destinations/cdpresolution/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "main": "./dist/cjs",
+  "module": "./dist/esm",
+  "scripts": {
+    "build": "yarn build:esm && yarn build:cjs",
+    "build:cjs": "tsc --module commonjs --outDir ./dist/cjs",
+    "build:esm": "tsc --outDir ./dist/esm"
+  },
+  "typings": "./dist/esm",
+  "dependencies": {
+    "@segment/browser-destination-runtime": "^1.4.0"
```
Comment (joe-ayoub-segment) [inline packages/browser-destinations/destinations/cdpresolution/package.json:19]:
```suggestion
    "@segment/browser-destination-runtime": "^1.12.2"
```

Hi @ajaysikri , could you update the version as specified please?

**Review by joe-ayoub-segment — CHANGES_REQUESTED** [review]

File: packages/browser-destinations/destinations/cdpresolution/package.json (hunk 1 above)
Comment (ajaysikri) [reply packages/browser-destinations/destinations/cdpresolution/package.json]: Hi Joe,

I have updated the version as requested.

Thanks, Ajay

**Comment by ajaysikri** [comment]
Hi @joe-ayoub-segment.  I have committed the suggested change.  Looking forward to any next steps.  Thanks, Ajay

**Comment by joe-ayoub-segment** [comment]
hi @ajaysikri apologies for the delay with this, I was out ill. 

There are still a few minor things which will need to be fixed before I can merge. 

The linter is complaining about the initialize function having unused params and variables. 

Also, a valid unit test will need to be added. 

Finally, could you add a description to the Destination please? 
Something like this: 

export const destination: BrowserDestinationDefinition<Settings, unknown> = {
  name: 'Cdpresolution',
  slug: 'actions-cdpresolution',
  `description: 'Sync Segment anonymousId to CDP Resolution',  `
  mode: 'device',

**Comment by joe-ayoub-segment** [comment]
Hi @ajaysikri just following up on this. Did you see my earlier comment above?
Best regards,
Joe

File: packages/browser-destinations/destinations/cdpresolution/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      ],
+      default: 'https://a.usbrowserspeed.com/cs',
+      required: true
+    }
+  },
+
+  initialize: async ({ settings, analytics }, deps) => {
+    //await deps.loadScript('<path_to_partner_script>')
+    // initialize client code here
+    const baseURL = settings.endpoint
```
[the remark this answers is not in the document]
Comment (ajaysikri) [reply packages/browser-destinations/destinations/cdpresolution/src/index.ts:53]: Added an empty return.  As per our last conversation we have no object to initialize.

**Review by joe-ayoub-segment — APPROVED** [review]

**Pull request merged**